docker/hapi-compose.yml

```diff
@@ -44,12 +47,15 @@ services:
     restart: unless-stopped
     ports:
       - 8091:8080
+    networks:
+      - cloudbuild
+      - default
       
 volumes:
   hapi-fhir-db:
   hapi-server:
 
 networks:
-  default:
-    external:
-      name: cloudbuild # Needed for Continuous integration
+  cloudbuild:
```

Comment:
Please do not hold this PR on this question: Do you know why we need this other network and why we can't use the default for continuous-integration too?
Adding @omarismail94 as I believe he first added this when working on e2e framework. @williamito please feel free to resolve and merge this if you don't know the answer.

Reply:
I don't know the reason. One possible reason is getting multiple configs onto the same default network by having them in the same folder is a useful side effect but probably not specifically intended, so sometimes things become brittle. I've found that using the `--force-recreate` flag fixes these issues though, and we already recommend using that flag when bringing up the test server docker configs.